src/App.tsx

```diff
@@ -0,0 +1,34 @@
+import "./App.css";
+import ForecastPage from "./Containers/ForecastPage";
+import HourlyForeCastPage from "./Containers/HourlyForeCastPage";
+import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
+import { WeatherStateData } from "./Context";
```

Comment:
Fix imports for all the files and keep them consistent, first would be imports form packages after those containers/components and after that styles/css

Reply:
ok